Automattic/pocket-casts-android
# Redesign chapter models

## Description

This streamlines chapter models.

* Removed unused properties and methods.
* Made the models immutable.
* Used `Duration` type to represent times of chapters.

## Testing Instructions

Do regression tests on chapters. You can use, for example, [this episode](https://pca.st/cb8an86i).

1. Make sure that chapters are displayed correctly.
2. Make sure that the playback progress is displayed on chapters.
3. With Patron account make sure that chapter selection still works.

## Checklist
 
#### I have tested any UI changes...
